src/adapters/openx.js

```diff
@@ -1,130 +1,228 @@
-          POX.addAdUnit(bids[i].params.unit);
+  function buildRequest(bids, params, delDomain) {
+    if (!utils.isArray(bids)) {
+      return;
+    }
+
+    params.auid = utils._map(bids, bid => bid.params.unit).join('%2C');
+    params.aus = utils._map(bids, bid => {
+      return utils.parseSizesInput(bid.sizes).join(',');
+    }).join('|');
+
+    bids.forEach(function (bid) {
+      for (let customParam in bid.params.customParams) {
+        if (bid.params.customParams.hasOwnProperty(customParam)) {
+          params["c." + customParam] = bid.params.customParams[customParam];
         }
+      }
+    });
+
+    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
+    let queryString = buildQueryStringFromParams(params);
 
-        POX.addHook(function (response) {
-          var i;
-          var bid;
-          var adUnit;
-          var adResponse;
-
-          // Map each bid to its response
-          for (i = 0; i < bids.length; i++) {
-            bid = bids[i];
-
-            // Get ad response
-            adUnit = response.getOrCreateAdUnit(bid.params.unit);
-            // If 'pub_rev' (CPM) isn't returned we got an empty response
-            if (adUnit.get('pub_rev')) {
-              adResponse = adResponse = bidfactory.createBid(1);
-
-              adResponse.bidderCode = 'openx';
-              adResponse.ad_id = adUnit.get('ad_id');
-              adResponse.cpm = Number(adUnit.get('pub_rev')) / 1000;
-
-              adResponse.ad = adUnit.get('html');
-              if(adUnit.get('deal_id') !== undefined) {
-                adResponse.dealId = adUnit.get('deal_id');
-              }
-
-              // Add record/impression pixel to the creative HTML
-              var recordPixel = OX.utils.template(response.getRecordTemplate(), {
-                medium: OX.utils.getMedium(),
-                rtype: OX.Resources.RI,
-                txn_state: adUnit.get('ts')
-              });
-              adResponse.ad += '<div style="position:absolute;left:0px;top:0px;visibility:hidden;"><img src="' + recordPixel + '"></div>';
-
-              adResponse.adUrl = adUnit.get('ad_url');
-              adResponse.width = adUnit.get('width');
-              adResponse.height = adUnit.get('height');
-
-              bidmanager.addBidResponse(bid.placementCode, adResponse);
-            } else {
-              // Indicate an ad was not returned
-              adResponse = bidfactory.createBid(2);
-              adResponse.bidderCode = 'openx';
-              bidmanager.addBidResponse(bid.placementCode, adResponse);
-            }
-          }
-        }, OX.Hooks.ON_AD_RESPONSE);
-
-        // Make request
-        POX.load();
-      }, true);
+    adloader.loadScript(`//${delDomain}/w/1.0/arj?${queryString}`);
+  }
+
+  function callBids(params) {
+    let isIfr = window.self !== window.top,
+      bids = params.bids || [],
+      currentURL = window.location.href && encodeURIComponent(window.location.href);
```

Comment:
Here we're trying to check if window.location.href exists to begin with, and if it is undefined we want to leave it as such instead of encoding it to be a string 'undefined'